ProcessLib/Process.cpp

```diff
@@ -149,7 +162,8 @@ void Process::setInitialConditions(const unsigned pcs_id, double const t,
     }
 }
 
-MathLib::MatrixSpecifications Process::getMatrixSpecifications() const
+MathLib::MatrixSpecifications Process::getMatrixSpecifications(
+    const int /*equation_id*/) const
```

Comment:
What is a difference between equation_id and process_id?

Reply:
Unified to use `process_id`.